astroplan/target.py

```diff
@@ -127,7 +127,7 @@ def from_name(cls, query_name, name=None, **kwargs):
         Examples
         --------
         >>> from astroplan import FixedTarget
-        >>> sirius = FixedTarget.from_name("Sirius")
+        >>> sirius = FixedTarget.from_name("Sirius")  # doctest: +REMOTE_DATA
```

Comment:
Problem with marking lines with remote data, you also need to mark all the subsequent lines derived from here onwards.

Reply:
Yes, there are some tests where I should mark also the next line

Reply:
I have also noticed that I was using REMOTE_DATA in some tests that don't need it. I have removed them